.gitignore

```diff
@@ -1,2 +1,3 @@
 /docs/
 /target/
+env/
```

Comment:
Careful that this will exclude *any* `env` folder in any sub-folder, not only at the repo's root. If the latter is the desired behavior, use `/env/` instead.